mappings/net/minecraft/world/gen/WorldProvider.mapping

```diff
@@ -1,6 +1,8 @@
 CLASS none/auf net/minecraft/world/gen/WorldProvider
 	FIELD b world Lnone/aiw;
 	FIELD c biomeProvider Lnone/ajn;
+	FIELD d waterEvaporates Z
+	FIELD e noSky Z
```

Comment:
Maybe rather than a negative variable name, `void` or something could be better. The getter in that case could be `isVoid`. (I'm assuming this value is true in end/nether worlds).